plugins/obs-ffmpeg/CMakeLists.txt

```diff
@@ -52,15 +52,15 @@ if(ENABLE_FFMPEG_LOGGING)
 		obs-ffmpeg-logging.c)
 endif()
 
+list(APPEND obs-ffmpeg_SOURCES
+	jim-nvenc.c
+	jim-nvenc-helpers.c)
+list(APPEND obs-ffmpeg_HEADERS
+	jim-nvenc.h)
```

Comment:
This might need to check for "windows or linux" or "not macOS"

Reply:
Yep, did that now